src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -55,6 +56,19 @@ public JsonNode listProjects(Path directory) throws Exception {
         mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
         ArrayNode arrayNode = mapper.createArrayNode();
 
+        // list projects
+        listProjects(directory, mapper, arrayNode);
```

Comment:
We should give this private method a different name to make it clear that its now the same method as the current one, I was wondering how it wasn't infinitely recursing at first. Maybe `listProjectsInDirectory`?